### PR TITLE
Update README.md with newly proposed view-model

### PR DESCRIPTION
**Pull request by PatrykSzwer**
After today's meeting with @warpech we decided to change the structure of fields inside `ItemMessages` object to not be arrays of messages, but only one message text with validation type. 

Also which is not part of this PR but would be good to discuss, we decided that it would be a better idea to instead of trying to dynamically create the structure of JSON for `ItemMessages`, have user to declare which properties should have validation structure during declaration.

### CODE REVIEW

**Comment by joozek78** [comment]
> have user to declare which properties should have validation structure during declaration.
what about collections? If I want to support `People.N.FirstName` where `N = 0,1,...` how do I declare that upfront?

**Comment by warpech** [comment]
> what about collections? If I want to support `People.N.FirstName` where `N = 0,1,...` how do I declare that upfront?

Each item in the collection has a precompiled JSON-subtree, so it's the same story:

```json
{
    "People": [{
        "FirstName$": "",
        "ItemMessages": {
            "FirstName": {
                "Message": "",
                "Invalid": ""
            }
        }
    }]
}
```

or am I missing something?

File: components/uni-form-item/README.md [hunk 1]
```diff
@@ -127,12 +127,12 @@ The following CSS Custom Properties are supported:
 
 ```json
 {
-    "FieldName": "",
+    "FieldName$": "",
     "ItemMessages": {
```
Comment (warpech) [inline components/uni-form-item/README.md]: Maybe call it `FormItemMetadata`?

**Comment by joozek78** [comment]
Do we really want to have separate `ItemMessages` for each subtree? It sounds cumbersome. I already hear developers asking why do they need 4 `ItemMessages` on one page. Also, that would require initializing all of them separately.
OTOH it may be inconvenient but more resilient. I can also imagine problems with reusing sub-view-models if we have just one `ItemMessages` on top

**Comment by PatrykSzwer** [comment]
I have some questions about `uni-form-item-group`. Currently proposed view-model for it is:
```
{
    "FieldGroupName": {
        "Label": "",
        "FirstItem": {"Label": "", "Value$": ""},
        "SecondItem": {"Label": "", "Value$": ""},
        "Invalid": "",
        "Message": ""
    }
}
```
`Invalid` and `message` will be easily replaced by `FormItemMetadata` but I'm not sure about `FirstItem` and `SecondItem`. Are those really needed? Client side can just put specific items inside `item-group`, I'm not sure if server side needs to be aware of this hierarchy. Messages can also be added directly for `FieldGroupName` if needed.

**Comment by warpech** [comment]
I agree that `FormItemMetadata` does not have to be hierarchical.

**Comment by PatrykSzwer** [comment]
I updated the view-model and example in `uni-form-item-group` part.

**Pull request merged**